roles/debian/hypervisor/tasks/main.yml

```diff
@@ -179,21 +109,6 @@
     state: restarted
   when: irqbalanceconf.changed
 
-- name: "systemd conf CPUAffinity"
-  lineinfile:
-    dest: /etc/systemd/system.conf
-    regexp: '^CPUAffinity=.*$'
-    line: "CPUAffinity={{ cpusystem }}"
-    state: present
-  when: cpusystem is defined
-- name: "systemd conf CPUAffinity revert"
-  lineinfile:
-    dest: /etc/systemd/system.conf
-    regexp: '^#?CPUAffinity=.*$'
-    line: "#CPUAffinity="
-    state: present
-  when: cpusystem is not defined
```

Comment:
Just a question here : 
How does Tuned interacts with the cgroups defined of SEAPATH Debian.
For example, here, you remove the CPUAffinity of the system slice, because it will be defined by Tuned.
But what about the other slices ? Like `machine.slice` for example ? Is it also handled by Tuned or do we still need to handle that externally ?

Reply:
systemd slices are not managed with tuned at all

Reply:
Oh, I confuse `system.conf` and the `system.slice`, my bad.

But anyway, would it be possible to manage slices with tuned ?
For example here, it seems to me that we defined the same thing twice : 
- `isolcpus` is used in `src/tuned/seapath-rt-host-variables.conf.j2`
- `cpusystem` is used in `system.slice` configuration.
To me, these two variables will always be the same, so why not define them in the tuned configuration

Reply:
They are not the same. These are different approaches: isolcpus vs cgroups/cpusets.

Tuned will not deal with slices/cgroups, so a standard use case will be to use tuned and not use slice at all. That way you isolate some cores, and use them in guests.
In that usecase you only have "isolated cores" and "non isolated cores", all slices have the same cpuaffinity (all cpus, but since you have isolcpus=domain... only "non isolated cores"), and you won't use cpusystem, cpuuser, cpumachines, etc. 

You can then choose you want to do housekeeping on specific cores, or run guests with a specific cpu affinity, in which case you will use cpusystem, cpumachinesrt, etc.
Like we discussed before, moving to slices/cpusets is linked to the isolcpus feature being deprecated, and seapath anticipating the future, but it's all optionnal for now.

This PR is really only about moving the isolcpus logic (with sysctl, sysfs, workqueues, etc.) to tuned. It does not involve the slices configuration which will still be managed by ansible (if used at all).

Reply:
Ok, thanks for the explanations